extensions/wikia/UserLogin/js/spec/MarketingOptIn.spec.js

```diff
@@ -0,0 +1,76 @@
+describe('UserLogin Marketing Opt In', function () {
+	'use strict';
+
+	var optIn,
+		geo,
+		wikiaFormMock;
+
+	beforeEach(function () {
+		geo = modules['wikia.geo']();
+		spyOn(geo, 'getContinentCode').andReturn('');
+		spyOn(geo, 'getCountryCode').andReturn('');
+
+		wikiaFormMock = {
+			inputs: {
+				wpMarketingOptIn: $('<input type="checkbox" class="hidden">')
+			},
+			getInputGroup: function () {
+				var $div = $('<div></div>');
+				$div.length = 1;
+				return $div;
+			}
+		};
+		wikiaFormMock.inputs.wpMarketingOptIn.length = 1;
```

Comment:
Do you need to include this line? I tried removing it and the tests pass fine without it.